### PR TITLE
Cosmetic changes to basicfwd.c

### PR DESCRIPTION
**Pull request by navaneethrvce**

### CODE REVIEW

**Comment by navaneethrvce** [comment]
Hi @KeithWiles , could you help take a look at this PR, 

Thanks!

File: examples/skeleton/basicfwd.c [hunk 1]
```diff
@@ -32,7 +32,7 @@ port_init(uint16_t port, struct rte_mempool *mbuf_pool)
 	const uint16_t rx_rings = 1, tx_rings = 1;
 	uint16_t nb_rxd = RX_RING_SIZE;
 	uint16_t nb_txd = TX_RING_SIZE;
-	int retval;
+	int retval = 0;
```
Comment (KeithWiles) [inline examples/skeleton/basicfwd.c:35]: You should not need to set this value to zero as line 45 sets retval anyway.

File: examples/skeleton/basicfwd.c [hunk 2]
```diff
@@ -102,7 +102,7 @@ port_init(uint16_t port, struct rte_mempool *mbuf_pool)
 	if (retval != 0)
 		return retval;
 
-	return 0;
+	return retval;
```
Comment (KeithWiles) [inline examples/skeleton/basicfwd.c:105]: At line 100 retval is set to zero or some value, but then at line 102 if retval != 0 we return retval, which means line 105 return 0 is ok too.

**Review by KeithWiles — COMMENTED** [review]
Not sure what you were trying to solve here, but I do not think it needs to be done unless I missed something.

File: examples/skeleton/basicfwd.c (hunk 2 above)
Comment (navaneethrvce) [reply examples/skeleton/basicfwd.c:105]: The code itself has no issues , but i feel returning a variable made the code more readable...

Comment (KeithWiles) [reply examples/skeleton/basicfwd.c:105]: I understand, but I believe the DPDK community will not see it that way IMO. As the only value that can be returned in that spot is zero :-(

Comment (navaneethrvce) [reply examples/skeleton/basicfwd.c:105]: Sure thing, Thanks for getting back on this!

**Pull request closed without merging**